class BankingError(Exception):
    pass

class NumberException(Exception):
    pass